e2e/src/usecases/Settings.js

```diff
@@ -23,11 +23,40 @@ export default Settings = () => {
     await element(by.id('ProfileEditName')).clearText()
     await element(by.id('ProfileEditName')).replaceText(`${randomName}`)
     await element(by.id('SaveButton')).tap()
-    await expect(element(by.text('Your name and picture were saved successfully.'))).toBeVisible()
+    await waitFor(element(by.text('Your name and picture were saved successfully.')))
+      .toBeVisible()
+      .withTimeout(1000 * 10)
     await dismissBanners()
     await waitForElementId('Hamburger')
     await element(by.id('Hamburger')).tap()
     // TODO replace this with an ID selector
     await expect(element(by.text(`${randomName}`))).toBeVisible()
   })
+
+  it('Change Language', async () => {
+    await element(by.id('ChangeLanguage')).tap()
+    await element(by.id('ChooseLanguage/es-419')).tap()
+    await waitFor(element(by.id('SettingsTitle')))
+      .toHaveText('Configuración')
+      .withTimeout(1000 * 15)
```

Comment:
Why does it need specific timeout?

Reply:
`waitFor` should have a timeout based on the [detox docs](https://github.com/nubank/detox/blob/master/docs/APIRef.waitFor.md#tobevisible).